api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -129,6 +141,7 @@ private class PurchasesAPI {
         val customerInfo4: CustomerInfo = purchases.awaitRestore()
         val customerInfo5: CustomerInfo = purchases.awaitSyncPurchases()
         var offerings: Offerings = purchases.awaitSyncAttributesAndOfferingsIfNeeded()
+        var consentStatus = purchases.getAmazonLWAConsentStatus()
```

Comment:
We should add the type here
```suggestion
        var consentStatus: AmazonLWAConsentStatus = purchases.getAmazonLWAConsentStatus()
```